### PR TITLE
Add protocolLocator to `AdvanceChannel` state

### PR DESCRIPTION
**Pull request by andrewgordstewart**

### CODE REVIEW

File: packages/wallet/src/redux/protocols/advance-channel/actions.ts [hunk 1]
```diff
@@ -18,7 +18,7 @@ export const clearedToSend: ActionConstructor<ClearedToSend> = p => {
 
 export function isAdvanceChannelAction(action: WalletAction): action is AdvanceChannelAction {
   return (
-    action.type === 'WALLET.ADVANCE_CHANNEL.COMMITMENTS_RECEIVED' ||
+    action.type === 'WALLET.COMMON.COMMITMENTS_RECEIVED' ||
```
Comment (lalexgap) [inline packages/wallet/src/redux/protocols/advance-channel/actions.ts]: Now that we're making use of the `ProtocolLocator` we can add a check for `ADVANCE_CHANNEL_PROTOCOL_LOCATOR`

File: packages/wallet/src/redux/protocols/advance-channel/reducer.ts [hunk 2]
```diff
@@ -154,6 +155,7 @@ function initializeWithNewChannel(
       nextParticipant(participants, ourIndex),
       processId,
       [signResult.signedCommitment],
+      ADVANCE_CHANNEL_PROTOCOL_LOCATOR,
```
Comment (lalexgap) [inline packages/wallet/src/redux/protocols/advance-channel/reducer.ts]: Shouldn't this be `protocolLocator` from state?

Comment (andrewgordstewart) [reply packages/wallet/src/redux/protocols/advance-channel/reducer.ts]: From what state?

Comment (lalexgap) [reply packages/wallet/src/redux/protocols/advance-channel/reducer.ts]: It should be the`protocolLocator` variable that is passed into `initialize`. That way the action that gets dispatched has the full `protocolLocator` string (IE:`DirectFunding-AdvanceChannel`) as opposed to just `AdvanceChannel.

**Pull request merged**